commands/host.vegman

```diff
@@ -224,6 +224,90 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
```

Comment:
I guess that this code is excess now as the `--yes` is already checked by the caller and passed to here only when it really specified by the user.
So, it seems to me, that `[[ -z "${1:-}" ]]` is enough to determine the confirmation is needed.